### PR TITLE
Run shellcheck

### LINKED ISSUE #69: Run shellcheck on CI for the deploy scripts stored here
cf https://github.com/whatwg/html-build/blob/32835201f72cf4518369065a6c20d2de50369f0c/.travis.yml

### PR DESCRIPTION
**Pull request by annevk**
Fixes #69.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,8 @@ env:
     - SERVER_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDP7zWfhJdjre9BHhfOtN52v6kIaDM/1kEJV4HqinvLP2hzworwNBmTtAlIMS2JJzSiE+9WcvSbSqmw7FKmNVGtvCd/CNJJkdAOEzYFBntYLf4cwNozCRmRI0O0awTaekIm03pzLO+iJm0+xmdCjIJNDW1v8B7SwXR9t4ElYNfhYD4HAT+aP+qs6CquBbOPfVdPgQMar6iDocAOQuBFBaUHJxPGMAG0qkVRJSwS4gi8VIXNbFrLCCXnwDC4REN05J7q7w90/8/Xjt0q+im2sBUxoXcHAl38ZkHeFJry/He2CiCc8YPoOAWmM8Vd0Ukc4SYZ99UfW/bxDroLHobLQ9Eh"
     - NEW_SERVER_PUBLIC_KEY="ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBDt6Igtp73aTOYXuFb8qLtgs80wWF6cNi3/AItpWAMpX3PymUw7stU7Pi+IoBJz21nfgmxaKp3gfSe2DPNt06l8="
 script:
+  - shellcheck deploy.sh
```
Comment (domenic) [inline .travis.yml]: Consider just `shellcheck **/*.sh`

File: deploy.sh [hunk 2]
```diff
@@ -1,21 +1,21 @@
 #!/bin/bash
 set -e
 
-if [ "$TRAVIS_BRANCH" != "master" -o "$TRAVIS_PULL_REQUEST" != "false" ]; then
+if [ "$TRAVIS_BRANCH" != "master" ] || [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```
Comment (domenic) [inline deploy.sh]: I think the best version is

```bash
if [[ "$TRAVIS_BRANCH" != "master" || "$TRAVIS_PULL_REQUEST" != "false" ]]
```

File: resources.whatwg.org/build/deploy.sh [hunk 3]
```diff
@@ -63,7 +63,7 @@ rm -rf "$WEB_ROOT" || exit 0
 
 copy_extra_files() {
     if [[ "$EXTRA_FILES" != "" ]]; then
-        # Will not pass ShellCheck: https://stackoverflow.com/q/45931553/3191
+        # shellcheck disable=SC2086
```
Comment (domenic) [inline resources.whatwg.org/build/deploy.sh]: Maybe keep the StackOverflow link

File: .travis.yml (hunk 1 above)
Comment (annevk) [reply .travis.yml]: Does not work.

Comment (domenic) [reply .travis.yml]: It really should; how strange. Oh well.

**Pull request merged**